### ISSUE TITLE
Setting initial selected date does not ignore time, causing `isSelected(day)` to incorrectly return false

### ISSUE THREAD

**Issue by veloware** [its-danny/use-lilius#492]
**Describe the bug**
Setting a default selected day, does not ignore the time portion of the date, meaning `isSelected(day)` returns false

**To Reproduce**

When setting the default selected date using - 

```
const {...} = useLilius({ selected: new Date()})
```

then rendering out days as per the chakra example - 

```
{calendar[0].map((week) => (
        <Box key={`week-${week[0]}`} sx={styles.calendarMatrixContainer}>
          {week.map((day) => (
              <Box
                data-in-range={inRange(day, startOfMonth(viewing), endOfMonth(viewing))}
                data-selected={isSelected(day)}
               ....
```

when it comes to the `isSelected(day)` for the box that represents 'today', it returns false, as `day` has its time set to 0, however the `selected` argument in the `useLilius({ selected: [ new Date() ]})` keeps the timestamp.

console output for 'selected' from `Returns` - 
<img width="486" alt="image" src="https://user-images.githubusercontent.com/61578546/173080643-1b164bcb-710d-4c0b-8ca4-218dd839ce28.png">

console output for `day` from `calendar` map
<img width="371" alt="image" src="https://user-images.githubusercontent.com/61578546/173080943-faf4216c-5089-4adb-b517-8abd1b8747a2.png">

However, manually stripping the time off before setting the initial selected date, seems to solve the issue - 

```
const {...} = useLilius({ selected: [ new Date(new Date().toDateString()) ]})
```

**Expected behavior**
Setting the default selected date using `new Date()` should ignore the time so it can accurately be compared with dates from the `calendar`

**Comment by its-danny** [comment]
thanks for the heads up, will get on this over the weekend. if you need this to work ASAP, in the meantime you can alternatively use `clearTime`:

```tsx
import { useLilius, clearTime } from "use-lilius";

const {...} = useLilius({ selected: [ clearTime(new Date()) ]);
```

**Comment by its-danny** [comment]
@veloware fixed in [2.0.1](https://www.npmjs.com/package/use-lilius/v/2.0.1)
